doc/developer_guide/api.rst

```diff
@@ -51,13 +51,13 @@ User Interface
 .. automodapi:: glue.qt.widgets.data_viewer
    :no-inheritance-diagram:
 
-.. automodapi:: glue.qt.widgets.scatter_widget
```

Comment:
random question -- why `viewer_widget` instead of `viewer` or `widget`? No strong opinion from me